### PR TITLE
README update

### PR DESCRIPTION
**Pull request by 0xtimmy**
- added instructions on how to run the test suite
- added instruction on including gibbon in PATH

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,9 +67,16 @@ At this point you can run the Gibbon executable:
...
     $ cabal v2-exec -w ghc-9.0.1 gibbon -- -h
 
-If you'd like to run the testsuite, you can do so with:
+And add gibbon to PATH
+```
+$ export PATH = "{ ... }/gibbon/dist-newstyle/build/{architecture}/ghc-{version}/gibbon-0.2/x/gibbon/build/gibbon/gibbon:$PATH"
+```
 
-    $ cd $GIBBONDIR && ./run_all_tests.sh
+If you'd like to run the testsuite, you can do so with:
```
Comment (ckoparkar) [inline README.md:75]: Did you run into a problem with the `run_all_tests` script? I'll admit that I haven't ran it in a while, but the goal is to keep it up-to-date.

File: README.md [hunk 2]
```diff
@@ -67,9 +67,16 @@ At this point you can run the Gibbon executable:
 
     $ cabal v2-exec -w ghc-9.0.1 gibbon -- -h
 
-If you'd like to run the testsuite, you can do so with:
+And add gibbon to PATH
+```
+$ export PATH = "{ ... }/gibbon/dist-newstyle/build/{architecture}/ghc-{version}/gibbon-0.2/x/gibbon/build/gibbon/gibbon:$PATH"
```
Comment (ckoparkar) [inline README.md:72]: It's possible that the dist-newstyle directory structure could change and we wouldn't notice. Did `v2-exec` not work for you?

**Review by ckoparkar — COMMENTED** [review]
Thank you for your PR and sorry for sleeping on it until now.

File: README.md (hunk 2 above)
Comment (vidsinghal) [reply README.md:72]: I think that path is just useful if you want to get the `gibbon` binary added to your PATH variable so its available on the terminal.

Comment (ulysses4ever) [reply README.md:72]: Hey there! Just FYI: you don’t need to spell the path to the binary explicitly if you leverage `cabal`’s `list-bin` subcommand, e.g.

```
$ export PATH = "$(cabal list-bin gibbon):$PATH"
```

**Comment by ulysses4ever** [comment]
The PR was closed automatically because the `master` branch it targeted was not active for months and was removed. Please, feel free to reopen the PR against the `main` branch (after rebasing).

**Pull request closed without merging**